Modules/System/Upgrade Tags/src/UpgradeTagImpl.Codeunit.al

```diff
@@ -29,6 +29,33 @@ codeunit 9996 "Upgrade Tag Impl."
         SetUpgradeTagForCompany(NewTag, CopyStr(CompanyName(), 1, MaxStrLen(ConstUpgradeTags.Company)));
     end;
 
+    procedure SetSkippedUpgrade(ExistingTag: Code[250]; SkipUpgrade: Boolean)
+    begin
+        SetSkippedUpgrade(ExistingTag, CompanyName(), SkipUpgrade);
```

Comment:
CompanyName is Text, parameter is Code[30]. Possible overflow.

Reply:
Changed